Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -1544,14 +1544,29 @@ namespace OpenInfraPlatform {
 						carve::geom3d::Vector& first = loop.front();
 						carve::geom3d::Vector& last = loop.back();

```

Comment:
This looks like unnecessary code.

Reply:
Yes - if you save a variable somewhere in order to avoid calling a function like `loop.front()` or `loop.back()` to always refer to the same variable, make sure that you also use the variable everywhere where you intend to use it :)

Otherwise saving a variable obtained from a function is good :)

Reply:
Resolved: 779b8e1